instructors.html

```diff
@@ -28,15 +28,22 @@
         <div class="col-md-10 col-md-offset-1">
                     <a href="index.html"><h1 class="title">Version Control with Mercurial</h1></a>
           <h2 class="subtitle">Instructor’s Guide</h2>
-          <h2 id="legend">Legend</h2>
-<p>Wolfman and Dracula have been hired by Universal Missions (a space services spin-off of Euphoric State University) to figure out where the company should send its next planetary lander. They want to be able to work on the plans at the same time, but they have run into problems doing this in the past. If they take turns, each one will spend a lot of time waiting for the other to finish, but if they work on their own copies and email changes back and forth things will be lost, overwritten, or duplicated.</p>
-<p>The solution is to use <a href="reference.html#version-control">version control</a> to manage their work. Version control is better than mailing files back and forth because:</p>
+          <p>Using a software tool to handle the versions of your project files lets you focus on the more interesting/innovative aspects of your project.</p>
+<ul>
+<li>Version control’s advantages:
 <ul>
 <li>Nothing that is committed to version control is ever lost. This means it can be used like the “undo” feature in an editor, and since all old versions of files are saved it’s always possible to go back in time to see exactly who wrote what on a particular day, or what version of a program was used to generate a particular set of results.</li>
```

Comment:
I think the clause "it’s always possible to go back in time to see exactly who wrote what on a particular day" is redundant with the next list item